### ISSUE TITLE
New subject prefix / new subject format

### ISSUE THREAD

**Issue by sp-aj** [cleidigh/EditEmailSubject-MX#55]
Hello,

We at South Pole believe that this plugin could be very useful for us. However there is a feature that we need to use the addon in production at our company.

The feature is the ability to set a prefix or smth like that in in the options page. Say that I get a email with the subject "test", then when I edit the subject the prefix, if one is set in the options page for example Date: ${date}, then Date: and todays date should be added to the beginning of the default new subject and saved if the user accepts it. Alternatively there could be option to set a format in options, smth like "$date $time $old_subject" instead of a prefix but our current fork uses the prefix version.

We have already implemented code that could be merged into your plugin in our local fork.
If you are alright with accepting this feature, I would be happy to make a pull request. I think it would benefit both us and other users of the plugin.
Thanks!

With kind regards, August
South Pole

**Comment by jobisoft** [comment]
Hi,

I will rewrite this add-on to use pure WebExtension APIs and will check back with you once that is done.

**Comment by sp-aj** [comment]
> Hi,
> 
> I will rewrite this add-on to use pure WebExtension APIs and will check back with you once that is done.

Hi,

Thanks for your reply.
Please do so, I look forward to it.

**Comment by jobisoft** [comment]
WebExtension rewrite is done. Code has changed a lot, see https://github.com/cleidigh/EditEmailSubject-MX/releases/tag/v7.2

Feel free to send me a pull request to add your feature for the v7.* branch.
